### DISCUSSION TITLE
“New Chat” and “Functionality Switch” buttons do not appear.

### DISCUSSION THREAD

**Discussion by luccamrofc** [danny-avila/LibreChat#5014] [Troubleshooting]
### What happened?

My system was working correctly, but today, upon logging in, the buttons disappeared.

I tested updating to a new version, as I thought it could help, but the problem not only persisted but worsened—the menus simply do not appear (see images).

An observation: I tried adjusting the Zoom, thinking it might be the issue, but in my tests, there seems to be no correlation.

![image](https://github.com/user-attachments/assets/11b0bdfd-068f-42c8-b8db-436dd4833c0a)

It seems that the issue is specific to Safari, as it works normally in other browsers.

### Steps to Reproduce

1- Open Safari
2-Open Your LibreChat
3-See the bug.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks for the report. I moved this to troubleshooting because this can't be reproduced.

On Safari:

![parsecd_GLIsMO83LZ](https://github.com/user-attachments/assets/c99764f9-6713-4c91-ad4c-a8e21e4e385c)

I would check your Safari version, see if anyone is having issues with other apps. I'm using Version 18.1.1 (20619.2.8.11.12) on Apple Macbook.
